setup/setup-ci/security-scanner/README.md

```diff
@@ -0,0 +1,181 @@
+---
+page_type: sample
+languages:
+- bash
+- python
+products:
+- azure-machine-learning
+description: Sample setup script to scan Compute Instances for malware and security vulnerabilities
+---
+
+# Compute Instance Security Scanner
+
+[![license: MIT](https://img.shields.io/badge/License-MIT-purple.svg)](../../../LICENSE)
+
+A security scanner for Azure ML [Compute Instances](https://learn.microsoft.com/en-us/azure/machine-learning/concept-compute-instance) reporting malware and vulnerabilities in OS and Python packages to [Azure Log Analytics](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-overview). For details on the vulnerability management process for the Azure Machine Learning service, see [Vulnerability Management](https://learn.microsoft.com/azure/machine-learning/concept-vulnerability-management).
+
+## Getting Started
+
+> Prerequisite: an Azure ML workspace with a Compute Instance running and [diagnostic logs](https://learn.microsoft.com/en-us/azure/machine-learning/monitor-azure-machine-learning) streaming to Log Analytics. See further down for alternative setups.
+
+- **Upload the scanner to Azure ML**: download [`amlsecscan.py`](amlsecscan.py), open [Azure ML Studio](https://ml.azure.com/), go to the Notebooks tab, and upload the file into your user folder `/Users/{user_name}` (replacing `{user_name}` with your user alias).
```

Comment:
Instead of bullets, these feel like steps, so maybe switch to numbered. For the first bullet, it seems like sub-steps, so instead of commas, break out into tab indented steps?

Reply:
Switched to number list and sub-steps